fleet_management_http_client_python/api/tenant_api.py

```
@@ -0,0 +1,826 @@
+    BringAuto Fleet Management v2 API
+
+    Specification for BringAuto fleet backend HTTP API
+
+    The version of the OpenAPI document: 4.0.0-alpha
+    Contact: fleet@bringauto.com
+    Generated by OpenAPI Generator (https://openapi-generator.tech)
+
+    Do not edit the class manually.
+"""  # noqa: E501
+
+import warnings
```

Ruff (0.8.2)

`warnings` imported but unused

Remove unused import: `warnings`

(F401)